raiden/tests/utils/migrations.py

```diff
@@ -0,0 +1,30 @@
+from hexbytes import HexBytes
+
+from raiden.tests.utils.factories import make_block_hash
+from raiden.utils.typing import Any, Dict, Tuple
+
+
+class FakeEth(object):
```

Comment:
afaik there is no need to derive from object, that's the default.

Reply:
In python 3, no need. This is a habit I got from python2, for the "new style" of classes back there. But I guess explicitness does not hurt.

Reply:
Please, please, let's just do things one way, yeah? Specially this things that make no difference what so ever.

```
egrep -e '^class [[:alnum:]]+\(object\):' $(find raiden -iname '*.py') | wc -l
3
egrep -e '^class [[:alnum:]]+:' $(find raiden -iname '*.py') | wc -l
55
```

There are 3 places where `class A(object)` is used and 55 where nothing is used.